specification/05-security_model.adoc

```diff
@@ -83,7 +83,7 @@ The CoVE-IO security model aims at protecting the above-described TVM assets
 from the following adversaries:
 
 * _Privileged host software adversary_: This includes host software components
-  executing in S and M mode like the host firmware, kernel, hypervisor, VMM,
+  executing in S/HS mode like the host firmware, kernel, hypervisor, VMM,
```

Comment:
Why are we removing M? Also host firmware may be an M component.

Reply:
M mode component is in TCB, right?
If so, it cannot be Privileged adversary.

Reply:
Not all M mode components are in the confidential computing TCB.

Reply:
Got you.

I am checking https://github.com/riscv-non-isa/riscv-ap-tee/blob/main/specification/threatmodel.adoc.

To align the definition, I believe "M mode component" should be "**Startup Software adversary**".

Thought?

Reply:
That makes perfect sense to me. We should align with the CoVE spec.

Reply:
Move the M-mode to Startup Adversary.